labrador/Cargo.toml

```diff
@@ -6,4 +6,3 @@ repository.workspace = true
 license.workspace = true
 
 [dependencies]
-base64 = "0.22.1"
```

Comment:
Could you please confirm whether the `labrador` crate should be part of a workspace or function as a standalone crate? I noticed the `workspace = true` declarations, which typically indicate a workspace setup. What's the intended direction for this?

Reply:
Good question! At the current stage we work in the workspace (aka monorepo) to easily propagate changes. As we plan to develop an aggregation scheme, it will go as a separate crate in this repo.

Probably in the future after stabilization and optimizations, we would like to set `labrador` as a separate repository